docs/user/check-ref/redis_wrapper.rst

```diff
@@ -10,6 +10,16 @@ Read-only access to Redis servers is provided by the :py:func:`redis` function.
     of the current entity's ``host`` attribute, and :samp:`{<port>}` is the given port (default ``6379``). See
     below for a list of methods provided by the returned connection object.
 
+    :param host: Redis host.
+    :type host: str
+
+    :param password:  If set - enables authentication to the destination server with the password provided.
+    :type password: str
+
+.. note::
+
+    You can also use ``plugin.redis.password`` plugin confirguration for zmon appliance to enable password auth for this plugin.
```

Comment:
ZMON appliance is a special form of deployment. This part is mainly concerned with Worker plugins. So I would rather mention that redis password can be configured.

Reply:
tnx, done, please take a look.